go.mod

```diff
@@ -1,36 +1,149 @@
 module github.com/airbrake/gobrake/v5
 
-go 1.16
+go 1.17
 
 require (
-	github.com/ajg/form v1.5.1 // indirect
 	github.com/apex/log v1.9.0
 	github.com/beego/beego/v2 v2.0.5
 	github.com/buger/jsonparser v1.1.1
 	github.com/caio/go-tdigest v3.1.0+incompatible
 	github.com/gin-gonic/gin v1.8.1
 	github.com/gobuffalo/buffalo v0.18.9
 	github.com/gofiber/fiber/v2 v2.36.0
-	github.com/google/go-querystring v1.1.0 // indirect
 	github.com/gorilla/mux v1.8.0
-	github.com/imkira/go-interpol v1.1.0 // indirect
 	github.com/jonboulle/clockwork v0.3.0
 	github.com/kataras/iris/v12 v12.1.8
 	github.com/labstack/echo/v4 v4.8.0
-	github.com/leesper/go_rng v0.0.0-20190531154944-a612b043e353 // indirect
-	github.com/moul/http2curl v1.0.0 // indirect
 	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.20.0
 	github.com/pkg/errors v0.9.1
 	github.com/rs/zerolog v1.27.0
-	github.com/smartystreets/goconvey v1.7.2 // indirect
 	github.com/urfave/negroni v1.0.0
 	github.com/valyala/fasthttp v1.39.0
+	go.uber.org/zap v1.17.0
+)
+
+require (
+	github.com/BurntSushi/toml v1.1.0 // indirect
```

Comment:
Why do we have so many new indirect dependencies?

Reply:
As we support many integrations and middlewares, they show up when we run `go mod tidy`.